### DISCUSSION TITLE
NPI and Facilities

### DISCUSSION THREAD

**Discussion by jschmor** [CMSgov/price-transparency-guide#273] [Q&A] [answered]
It has been my experience that many contracts and pricing tables are NOT consistently linked by NPI for facilities. Many times for facilities the CMS Certification Number (CCN) (legacy name OSCAR) is the unique facility ID for Inpatient, Outpatient, and ASC facilities. Is anyone else experiencing this issue with facilities and is it something that might be worth considering as an ID to capture for institutional types?

Source of the CCN ids:
https://data.cms.gov/provider-characteristics/hospitals-and-other-facilities/provider-of-services-file-hospital-non-hospital-facilities

**Comment by ryan-thomas-peterson** [comment]
For institutional claims, we have the same experience @jschmor.  Contracts we have are often written at the TIN level and not maintained for every office site.  Entity type 2 NPIs (facilities instead of practitioners) are not cleanly defined.  CMS guidance is that each subpart of an organization may designate their own NPI, but what a subpart is more loosely defined (https://www.cms.gov/Regulations-and-Guidance/Administrative-Simplification/NationalProvIdentStand/Downloads/npi_fs_subparts_032106.pdf).  We have found NPIs and subparts for facilities come and go and for many contracts we don't actively maintain this level of granularity for contract implementation.  

For institutional billing types, what should be listed in the provider array?  Should it be the entity type 2 NPIs?  If so, can documentation be updated?  

If contracts do not require designation of an NPI what other approaches are people considering?

**Reply by rgouldkc** [reply]
For now since the documentation states Type 1 NPIs only all I am reporting for facilities is the TIN and leaving the NPIs empty

**Reply by ryan-thomas-peterson** [reply]
@rgouldkc, this has been our approach as well.

**Reply by taylorpatriciab** [reply]
@rgouldkc @ryan-thomas-peterson Is leaving NPI empty compliant with the schema?  NPI is a required data element

**Reply by taylorpatriciab** [reply]
@jschmor @ryan-thomas-peterson The schema requires Type 1 NPIs, which are providers who are individuals, including physicians, dentists, and all sole proprietors.  Facilities are assigned Type 2 NPIs.  As I read it, NPIs are not required for facilities.

**Reply by BonHadVan** [reply]
Since NPI is a required element in the schema our understanding is that it cannot be empty.  We intended to report the Type 2 NPI's in the array for our Facility providers which appears in conflict with the 'definition' however there appears to be no editing that will prevent us from doing this.  CMS should provide clarity on the expectations for the NPI for Facility and Ancillary providers that don't have Type 1 NPI's.

**Reply by taylorpatriciab** [reply]
@shaselton-usds Could clarity on the expectations on populating the NPI for  Facility and Ancillary providers that don't have a Type 1 NPI be provided?

**Reply by skochin** [reply]
I believe there is still a need for guidance on identifying facilities in the files - as mentioned, Type 1 NPIs can only be held by people, and we'd be missing (or obscuring) a significant part of the transparency equation if we don't identify rates paid to hospitals, ambulatory surgical centers, independent laboratories, and such.  @shaselton-usds is there an update on this?

**Reply by BonHadVan** [reply]
@shaselton-usds - Could you please clarify expectations for NPI when the provider is a Facility that only has a Type '2' NPI?  Should no NPI be sent?  Or should the Type 2 NPI be sent?

https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#providers-object

**Reply by shaselton-usds** [reply]
There is a PR that updates the documentation to clarify in the allowance of NPI Type 2 reporting here: https://github.com/CMSgov/price-transparency-guide/pull/446

**Comment by tdfow** [comment]
I am not seeing any reference to any 'Types of NPI' in the mandate.  Where in the mandate does it specify the type of NPI that should only be used?   It seems to only mention that NPI is required for the information being published.   https://www.cms.gov/CCIIO/Resources/Regulations-and-Guidance/Downloads/CMS-Transparency-in-Coverage-9915F.pdf

**Reply by skochin** [reply]
I believe you're correct, I have not found that specification in the rules either. NPI is a required data element for all providers, so it doesn't seem to make sense to limit the type of NPI used,

**Reply by ryan-thomas-peterson** [reply]
@tdfow the schema description under in-network rates in the Readme.md lists the following for npi under the provider object: "An array of individual (type 1) provider identification numbers (NPI).".  That was part of my initial confusion above that remains.

**Comment by rakeshkatte** [comment]
Since there is no mention of the type of NPI we are not differentiating
between the NPIs.

**Comment by shaselton-usds** [comment, marked as answer]
Updated the documentation to clarify the allowance of using NPI type 2s in reporting: https://github.com/CMSgov/price-transparency-guide/pull/446. Thanks for the patience:
